### PR TITLE
estate_agent.json - adding missing brand:wikidata

### PR DESCRIPTION
**Pull request by UKChris-osm**
Trying out my first fork / pull request - added **brand:wikidata** to _/brand/office/estate_agent.json_ for:

- Bairstow Eves
- Harcourts
- Ray White
- Your Move

### CODE REVIEW

File: brands/office/estate_agent.json [hunk 1]
```diff
@@ -107,6 +108,7 @@
   "office/estate_agent|Harcourts": {
     "tags": {
       "brand": "Harcourts",
+      "brand:wikidata": "Q5655056",
```
Comment (bhousel) [inline brands/office/estate_agent.json]:
```suggestion
      "brand:wikidata": "Q5655056",
      "brand:wikipedia": "en:Harcourts International",
```

**Comment by UKChris-osm** [comment]
Have you made this edit to my forked pull request? And do I have to okay it to be committed to this pull request? Still very new at this :)

**Comment by bhousel** [comment]
> Have you made this edit to my forked pull request? And do I have to okay it to be committed to this pull request? Still very new at this :)

It looks like the suggested change was added!  I think you might have clicked a button to accept it.  
Looks great, thanks for these additions 👍

**Comment by UKChris-osm** [comment]
Just hoping I do everything correctly :) So with this merge, does that mean this edit is all done and live on the master copy?

**Comment by bhousel** [comment]
> So with this merge, does that mean this edit is all done and live on the master copy?

Yes!  I just did `git pull` on my end and I see your change now on my copy too.

**Comment by UKChris-osm** [comment]
Excellent, thank you for doing it so quickly @bhousel. How often does the [iD Editor](http://ideditor.com) and [nsi.guide](https://nsi.guide/) get updated with the latest master?

**Comment by bhousel** [comment]
> How often does the [iD Editor](http://ideditor.com) and [nsi.guide](https://nsi.guide/) get updated with the latest master?

iD Editor gets updated whenever they have other things to release.. It tends to be about once a month.

https://nsi.guide gets updated whenever I run the command to do it.  So I'll do that now, and it will show the updates in about 10 minutes.

**Comment by UKChris-osm** [comment]
Thanks for running it, I see it live now on [nsi.guide](https://nsi.guide). This has been a good little learning curve for me, hopefully I'll be able to add more to the project over time :)

**Pull request merged**